distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -17,73 +17,4 @@
-log4j.appender.syslog.facility=AUTH
-# this should be set to loghost generally when a syslog host is configured
-log4j.appender.syslog.syslogHost=localhost
-
-# CONSOLE appender not used by default
-log4j.appender.stdout=org.apache.log4j.ConsoleAppender
-log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
-log4j.appender.stdout.layout.ConversionPattern=%d{ABSOLUTE} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
-
-# File appender
-log4j.appender.out=org.apache.log4j.RollingFileAppender
-log4j.appender.out.layout=org.apache.log4j.PatternLayout
-log4j.appender.out.layout.ConversionPattern=%d{ISO8601} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
-log4j.appender.out.file=${karaf.data}/log/ddf.log
-log4j.appender.out.append=true
-log4j.appender.out.maxFileSize=20MB
-log4j.appender.out.maxBackupIndex=10
-
-# Ingest Error File appender
-log4j.appender.ingestError=org.apache.log4j.RollingFileAppender
-log4j.appender.ingestError.layout=org.apache.log4j.PatternLayout
-log4j.appender.ingestError.layout.ConversionPattern=%d{ISO8601} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
-log4j.appender.ingestError.file=${karaf.data}/log/ingest_error.log
-log4j.appender.ingestError.append=true
-log4j.appender.ingestError.maxFileSize=20MB
-log4j.appender.ingestError.maxBackupIndex=10
-
-# Security File appender
-log4j.appender.security=org.apache.log4j.RollingFileAppender
-log4j.appender.security.layout=org.apache.log4j.PatternLayout
-log4j.appender.security.layout.ConversionPattern=[%-5p] %d{ISO8601} | %-16.16t | %-15.20c{1} |  %m%n
-log4j.appender.security.file=${karaf.data}/log/security.log
-log4j.appender.security.append=true
-log4j.appender.security.maxFileSize=20MB
-log4j.appender.security.maxBackupIndex=10
-
-# Sift appender
-log4j.appender.sift=org.apache.log4j.sift.MDCSiftingAppender
-log4j.appender.sift.key=bundle.name
-log4j.appender.sift.default=karaf
-log4j.appender.sift.appender=org.apache.log4j.FileAppender
-log4j.appender.sift.appender.layout=org.apache.log4j.PatternLayout
-log4j.appender.sift.appender.layout.ConversionPattern=%d{ABSOLUTE} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %m%n
-log4j.appender.sift.appender.file=${karaf.data}/log/$\\{bundle.name\\}.log
-log4j.appender.sift.appender.append=true
-
-# CXF and Solr logging is verbose.  Default setting to WARN.  This can be changed in the karaf console.
-log4j.logger.org.apache.cxf = WARN
-log4j.logger.org.apache.lucene = WARN
-log4j.logger.org.apache.solr = WARN
-log4j.logger.lux.solr = WARN
-log4j.logger.org.ops4j.pax.web.jsp = WARN
-log4j.logger.org.apache.aries.spifly = WARN
-log4j.logger.org.apache.cxf.phase.PhaseInterceptorChain = ERROR
+org.ops4j.pax.logging.log4j2.config.file = ${karaf.etc}/log4j2.config.xml
```

Comment:
This is JB's instruction for moving to log4j2 so I didn't try doing it differently.